messenger.rst

```diff
@@ -643,6 +643,25 @@ You can limit the worker to only process messages from specific queue(s):
 To allow using the ``queues`` option, the receiver must implement the
 :class:`Symfony\\Component\\Messenger\\Transport\\Receiver\\QueueReceiverInterface`.
 
+.. _messenger-message-count:
+
+Knowing the number of messages in the "queue" for transport(s)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to know how many messages are in the "queues",
```

Comment:
Begin with a version added directive for version 6.2

Reply:
Thanks, I had forgotten